frontend/src/metabase-lib/metadata/Table.unit.spec.ts

```diff
@@ -1,8 +1,16 @@
-import { PRODUCTS } from "__support__/sample_database_fixture";
+import { createMockMetadata } from "__support__/metadata";
+import {
+  createSampleDatabase,
+  PRODUCTS_ID,
+} from "metabase-types/api/mocks/presets";
 import Table from "./Table";
 
 describe("Table", () => {
-  const productsTable = new Table(PRODUCTS);
+  const metadata = createMockMetadata({
+    databases: [createSampleDatabase()],
+  });
+
+  const productsTable = metadata.table(PRODUCTS_ID) as Table;
```

Comment:
@ranquild yeah, I haven't used it here because we forbid imports from the app folder inside `metabase-lib` (and `checkNotNull` is under `metabase/core`)